pkg/generators/openapi.go

```diff
@@ -376,8 +415,9 @@ func (g openAPITypeWriter) generate(t *types.Type) error {
 				"SchemaProps: spec.SchemaProps{\n", args)
 			g.generateDescription(t.CommentLines)
 			g.Do("Type:$.type|raw${}.OpenAPISchemaType(),\n"+
-				"Format:$.type|raw${}.OpenAPISchemaFormat(),\n"+
-				"},\n"+
+				"Format:$.type|raw${}.OpenAPISchemaFormat(),\n", args)
+			g.generateValueValidations(&overrides.ValueValidations)
```

Comment:
Each case that adds a new `g.generateValueValidations` should have an associated unit test to show this case is being rendered properly.

existing nit: this code is kinda messy, a good future PR might clean this logic up a bit